### PR TITLE
Add Remote.closeCurrentPathFind function, so current pathfind can be …

### PR DESCRIPTION
**Pull request by darkdarkdragon**
…properly closed,

so new can be created without adding to queue

### CODE REVIEW

**Comment by clark800** [comment]
I think this should work. We should also throw an error if the user calls `createPathFind` without a callback with there is a `_cur_path_find`.

**Comment by darkdarkdragon** [comment]
@clark800 done

File: src/core/remote.js [hunk 1]
```diff
@@ -1822,6 +1835,10 @@ Remote.prototype.findAccount = function(accountID) {
  */
 Remote.prototype.createPathFind = function(options, callback) {
   if (this._cur_path_find !== null) {
+    if (callback === undefined) {
```
Comment (clark800) [inline src/core/remote.js]: I just thought of maybe a better idea: I think we can just skip this whole queuing block if the `callback` is undefined. That would better preserve the old behavior.

Comment (darkdarkdragon) [inline src/core/remote.js]: You mean is to create new `PathFind`, set it to `_cur_path_find` and return it?
The only problem with that, is what in `this._queued_path_finds` can be left hanging some requests.

Comment (clark800) [inline src/core/remote.js]: Yeah, maybe that approach would get too complicated. I think we should change the error message though. "Only one streaming pathfind request at a time is supported"

**Comment by darkdarkdragon** [comment]
@clark800 done

**Comment by clark800** [comment]
LGTM

**Pull request merged**